docs/usage/getting-started/use-cases.md

```diff
@@ -58,8 +58,8 @@ This can be because:
 - The package manager/file format is not supported, or
 - The file format is not a standard or is proprietary
 
-If your dependencies are not found by default, you can use our "regex" manager to set your own custom patterns to extract dependencies.
-You configure the regex manager by telling it:
+If your dependencies are not found by default, you can use our custom manager to set your own custom patterns to extract dependencies.
+You configure the custom manager by telling it:
```

Comment:
Needs quotation marks or backticks. @HonkingGoose wdyt?

Reply:
Yeah, just saying custom manager is hard to understand.

`customManager` seems good, if needed you can link to the `customManager` docs.

Reply:
Technically the term is `customManagers` plural. Like with `packageRules` we sometimes twist ourselves into knots about what to call a singular.

I would go with `custom` manager if in doubt, to indicate that it's a technical term and not just a choice of words